Changing NAT rule attributes

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior and the problem section from the TF file, without sensitive information.
1) Create rule
```
 resource "routeros_ip_firewall_nat" "test" {
   action      = "dst-nat"
   chain       = "dstnat"
   comment     = "Test rule"
   disabled    = true
   dst_address = "1.2.3.4"
   dst_port     = 12345
   protocol     = "tcp"
   to_addresses = "1.2.3.4"
   to_ports     = "22"
 }
```
2) Change rule
```
 resource "routeros_ip_firewall_nat" "test" {
   action      = "dst-nat"
   chain       = "dstnat"
   comment     = "Test rule"
   disabled    = true
   in_interface = "ether1"
   dst_port     = 12345
   protocol     = "tcp"
   in_interface = "wan1.ether1"
   to_addresses = "1.2.3.4"
   to_ports     = "22"
 }
```
**Expected behavior**
A clear and concise description of what you expected to happen.
```
Terraform will perform the following actions:

  # routeros_ip_firewall_nat.test will be updated in-place
  ~ resource "routeros_ip_firewall_nat" "test" {
      - dst_address  = "1.2.3.4" -> null
        id           = "*A"
      + in_interface = "wan1.ether1"
        # (10 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```
The dst_address field should be deleted, and the in_interface field should be added instead.

**Debug Information**
If needed, be ready to publish debug output of the provider (green lines after executing `TF_LOG=debug ROS_LOG_COLOR=1 terraform <command>`).
```
2025-05-22T16:31:40.721+0300 [INFO]  Terraform version: 1.10.1
2025-05-22T16:31:40.721+0300 [DEBUG] using github.com/hashicorp/go-tfe v1.70.0
2025-05-22T16:31:40.721+0300 [DEBUG] using github.com/hashicorp/hcl/v2 v2.23.0
2025-05-22T16:31:40.721+0300 [DEBUG] using github.com/hashicorp/terraform-svchost v0.1.1
2025-05-22T16:31:40.721+0300 [DEBUG] using github.com/zclconf/go-cty v1.15.1-0.20241111215639-63279be090d7
2025-05-22T16:31:40.721+0300 [INFO]  Go runtime version: go1.23.3
2025-05-22T16:31:40.721+0300 [INFO]  CLI args: []string{"terraform", "apply", "local_path"}
2025-05-22T16:31:40.721+0300 [DEBUG] Attempting to open CLI config file: /home/dobriiezh/.terraformrc
2025-05-22T16:31:40.721+0300 [INFO]  Loading CLI configuration from /home/dobriiezh/.terraformrc
2025-05-22T16:31:40.721+0300 [DEBUG] Explicit provider installation configuration is set
2025-05-22T16:31:40.721+0300 [INFO]  CLI command args: []string{"apply", "local_path"}
2025-05-22T16:31:40.747+0300 [DEBUG] checking for provisioner in "."
2025-05-22T16:31:40.748+0300 [DEBUG] checking for provisioner in "/usr/bin"
2025-05-22T16:31:40.748+0300 [INFO]  backend/local: starting Apply operation
2025-05-22T16:31:40.749+0300 [DEBUG] POST https://gitlab.ecom.tech/api/v4/projects/315/terraform/state/p02-oobgw-1/lock
2025-05-22T16:31:40.899+0300 [DEBUG] GET https://gitlab.ecom.tech/api/v4/projects/315/terraform/state/p02-oobgw-1
2025-05-22T16:31:41.049+0300 [DEBUG] created provider logger: level=debug
2025-05-22T16:31:41.049+0300 [INFO]  provider: configuring client automatic mTLS
2025-05-22T16:31:41.059+0300 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 args=[".terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0"]
2025-05-22T16:31:41.059+0300 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 pid=173166
2025-05-22T16:31:41.059+0300 [DEBUG] provider: waiting for RPC address: plugin=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0
2025-05-22T16:31:41.063+0300 [INFO]  provider.terraform-provider-routeros_v1.85.0: configuring server automatic mTLS: timestamp="2025-05-22T16:31:41.063+0300"
2025-05-22T16:31:41.077+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: plugin address: address=/tmp/plugin1928618654 network=unix timestamp="2025-05-22T16:31:41.076+0300"
2025-05-22T16:31:41.077+0300 [DEBUG] provider: using plugin: version=5
2025-05-22T16:31:41.108+0300 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-05-22T16:31:41.109+0300 [INFO]  provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 id=173166
2025-05-22T16:31:41.109+0300 [DEBUG] provider: plugin exited
2025-05-22T16:31:41.110+0300 [INFO]  backend/local: apply calling Apply
2025-05-22T16:31:41.110+0300 [DEBUG] Building and walking apply graph for NormalMode plan
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_firewall_addr_list_common.routeros_ip_firewall_addr_list.list (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.dhcp_server.routeros_ip_dhcp_server.server (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_firewall_nat.routeros_ip_firewall_nat.rule (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "routeros_ip_firewall_nat.test (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.dhcp_server_network.routeros_ip_dhcp_server_network.dhcp_server_network (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.interface_list_members.routeros_interface_list_member.list_member (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.dns_server.routeros_dns.dns-server (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.bridge_ports.routeros_interface_bridge_port.bridge_port (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.snmp_community.routeros_snmp_community.community (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "routeros_ip_firewall_raw.raw (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_services.routeros_ip_service.ip_service (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "routeros_ip_firewall_nat.test" (*terraform.NodeApplyableResourceInstance) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.bridges.routeros_interface_bridge.bridge (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.snmp.routeros_snmp.snmp (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.interface_list.routeros_interface_list.list (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_pool.routeros_ip_pool.pool (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ethernet.routeros_interface_ethernet.ethernet (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_address.routeros_ip_address.address (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_route.routeros_ip_route.route (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.ip_firewall_addr_list_specify.routeros_ip_firewall_addr_list.list (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.111+0300 [DEBUG] ProviderTransformer: "module.identity.routeros_system_identity.identity (expand)" (*terraform.nodeExpandApplyableResource) needs provider["registry.terraform.io/terraform-routeros/routeros"]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.bridges (expand)" references: [module.identity (expand) module.identity (close) module.ethernet (close) module.ethernet (expand) local.interface_bridge (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_specify (expand)" references: [module.identity (expand) module.identity (close) local.ip_firewall_addr_list (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.routeros_dns.dns-server (expand)" references: [module.dns_server.var.max_concurrent_tcp_sessions (expand) module.dns_server.var.servers (expand) module.dns_server.var.allow_remote_requests (expand) module.dns_server.var.query_server_timeout (expand) module.dns_server.var.doh_max_server_connections (expand) module.dns_server.var.doh_timeout (expand) module.dns_server.var.max_concurrent_queries (expand) module.dns_server.var.cache_max_ttl (expand) module.dns_server.var.doh_max_concurrent_queries (expand) module.dns_server.var.cache_size (expand) module.dns_server.var.address_list_extra_time (expand) module.dns_server.var.mdns_repeat_ifaces (expand) module.dns_server.var.verify_doh_cert (expand) module.dns_server.var.use_doh_server (expand) module.dns_server.var.max_udp_packet_size (expand) module.dns_server.var.vrf (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ethernet.var.logical_name (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "local.snmp_community (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server (expand)" references: [module.identity (expand) module.identity (close) module.ethernet (close) module.ethernet (expand) module.bridges (expand) module.bridges (close) module.ip_pool (expand) module.ip_pool (close) module.ip_address (expand) module.ip_address (close) local.dhcp_server (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.interface_list (expand)" references: [module.bridges (expand) module.bridges (close) module.ethernet (close) module.ethernet (expand) local.interface_list (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.allow_remote_requests (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server_network (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.identity (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.common_addr_list (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.identity.var.identity (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.engine_id_suffix (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp_community.var.community_config (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.common_addr_list.output.ip_firewall_addr_list (expand)" references: [module.common_addr_list.local.ip_firewall_addr_list (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_route (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.interface_list_members (expand)" references: [module.interface_list (expand) module.interface_list (close) local.interface_list (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.src_address (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "local.interface_list (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ethernet (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server_network (expand)" references: [module.dhcp_server (expand) module.dhcp_server (close) local.dhcp_server_network (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.bridge_ports.routeros_interface_bridge_port.bridge_port (expand)" references: [module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand) module.bridge_ports.var.bridge_port_config (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "local.ip_route (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.common_addr_list (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.vrf (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ethernet (expand)" references: [module.identity (expand) module.identity (close) local.ethernet_ports (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_pool (expand)" references: [module.identity (expand) module.identity (close) module.ethernet (close) module.ethernet (expand) module.bridges (expand) module.bridges (close) local.ip_pool (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_common.routeros_ip_firewall_addr_list.list (expand)" references: [module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp.routeros_snmp.snmp (expand)" references: [module.snmp.var.src_address (expand) module.snmp.var.trap_target (expand) module.snmp.var.trap_community (expand) module.snmp.var.enabled (expand) module.snmp.var.trap_generators (expand) module.snmp.var.engine_id_suffix (expand) module.snmp.var.trap_version (expand) module.snmp.var.trap_interfaces (expand) module.snmp.var.vrf (expand) module.snmp.var.location (expand) module.snmp.var.contact (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.bridge_ports (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.interface_list.routeros_interface_list.list (expand)" references: [module.interface_list.var.interface_list_config (expand) module.interface_list.var.interface_list_config (expand) module.interface_list.var.interface_list_config (expand) module.interface_list.var.interface_list_config (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.trap_target (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.address_list_extra_time (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.interface_list_members.output.member_ids (expand)" references: [module.interface_list_members.routeros_interface_list_member.list_member (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.output.dynamic_servers (expand)" references: [module.dns_server.routeros_dns.dns-server (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_nat (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_route (expand)" references: [module.ip_address (expand) module.ip_address (close) local.ip_route (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_pool.routeros_ip_pool.pool (expand)" references: [module.ip_pool.var.ip_pool_config (expand) module.ip_pool.var.ip_pool_config (expand) module.ip_pool.var.ip_pool_config (expand) module.ip_pool.var.ip_pool_config (expand)]
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_services.var.ip_service_config (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.interface_list_members (close)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "routeros_ip_firewall_nat.test (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "local.interface_bridge (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.mdns_repeat_ifaces (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.max_concurrent_queries (expand)" references: []
2025-05-22T16:31:41.113+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_nat (expand)" references: [module.ip_firewall_addr_list_common (close) module.ip_firewall_addr_list_common (expand) module.ip_firewall_addr_list_specify (expand) module.ip_firewall_addr_list_specify (close) local.nat_rules (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server (expand)" references: [module.identity (expand) module.identity (close)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.identity.routeros_system_identity.identity (expand)" references: [module.identity.var.identity (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server.routeros_ip_dhcp_server.server (expand)" references: [module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand) module.dhcp_server.var.dhcp_server_config (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "var.terraform_username" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.contact (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server.output.cache_used (expand)" references: [module.dns_server.routeros_dns.dns-server (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server (close)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_pool (close)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ethernet.routeros_interface_ethernet.ethernet (expand)" references: [module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.logical_name (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand) module.ethernet.var.interface_config (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_route.routeros_ip_route.route (expand)" references: [module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand) module.ip_route.var.ip_route_config (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_address.var.ip_address_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_common.var.ip_firewall_addr_list_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.enabled (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "routeros_ip_firewall_nat.test" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.bridges.routeros_interface_bridge.bridge (expand)" references: [module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand) module.bridges.var.bridge_config (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server.var.dhcp_server_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.bridge_ports (expand)" references: [module.bridges (expand) module.bridges (close) local.interface_bridge (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.interface_list (close)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ethernet.var.interface_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "routeros_ip_firewall_raw.raw (expand)" references: [module.ip_firewall_addr_list_common (close) module.ip_firewall_addr_list_common (expand) module.ip_firewall_addr_list_specify (expand) module.ip_firewall_addr_list_specify (close)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.interface_list_members.routeros_interface_list_member.list_member (expand)" references: [module.interface_list_members.var.interface_list_members_config (expand) module.interface_list_members.var.interface_list_members_config (expand) module.interface_list_members.var.interface_list_members_config (expand) module.interface_list_members.var.interface_list_members_config (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.doh_max_server_connections (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.verify_doh_cert (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_pool.var.ip_pool_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_common (close)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp_community (expand)" references: [module.identity (expand) module.identity (close) local.snmp_community (expand)]
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.trap_version (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.trap_generators (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.bridge_ports.var.bridge_port_config (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.max_udp_packet_size (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.identity (close)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.trap_community (expand)" references: []
2025-05-22T16:31:41.114+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server_network.var.dhcp_server_network_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.interface_list_members.var.interface_list_members_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "local.nat_rules (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_common (expand)" references: [module.identity (expand) module.identity (close) module.common_addr_list.output.ip_firewall_addr_list (expand)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.bridges (close)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "local.ip_pool (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "local.ip_services (expand)" references: [module.common_addr_list.output.ip_firewall_addr_list (expand) local.ip_firewall_addr_list (expand) module.common_addr_list.output.ip_firewall_addr_list (expand) local.ip_firewall_addr_list (expand) module.common_addr_list.output.ip_firewall_addr_list (expand) local.ip_firewall_addr_list (expand) module.common_addr_list.output.ip_firewall_addr_list (expand) local.ip_firewall_addr_list (expand)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_route.var.ip_route_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.max_concurrent_tcp_sessions (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.doh_max_concurrent_queries (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_address (expand)" references: [module.ethernet (close) module.ethernet (expand) module.bridges (expand) module.bridges (close) local.ip_address (expand)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_address (close)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_services.routeros_ip_service.ip_service (expand)" references: [module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand) module.ip_services.var.ip_service_config (expand)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_nat.var.nat_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server (close)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.interface_list.var.interface_list_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_specify (close)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_nat.routeros_ip_firewall_nat.rule (expand)" references: [module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand) module.ip_firewall_nat.var.nat_config (expand)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.doh_timeout (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "local.ip_firewall_addr_list (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.snmp_community (close)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.location (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.query_server_timeout (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "provider[\"registry.terraform.io/terraform-routeros/routeros\"]" references: [var.terraform_password var.terraform_username]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "var.terraform_password" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.cache_max_ttl (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.servers (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "local.dhcp_server (expand)" references: []
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.snmp (expand)" references: [module.snmp_community (expand) module.snmp_community (close)]
2025-05-22T16:31:41.115+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.cache_size (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "local.ethernet_ports (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.ip_services (expand)" references: [module.identity (expand) module.identity (close) module.ethernet (close) module.ethernet (expand) module.bridges (expand) module.bridges (close) module.ip_pool (expand) module.ip_pool (close) module.ip_address (expand) module.ip_address (close) local.ip_services (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.dhcp_server_network.routeros_ip_dhcp_server_network.dhcp_server_network (expand)" references: [module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand) module.dhcp_server_network.var.dhcp_server_network_config (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.ip_firewall_addr_list_specify.routeros_ip_firewall_addr_list.list (expand)" references: [module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand) module.ip_firewall_addr_list_specify.var.ip_firewall_addr_list_config (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.trap_interfaces (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "local.dhcp_server_network (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "local.ip_address (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.common_addr_list.local.ip_firewall_addr_list (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.dns_server.output.id (expand)" references: [module.dns_server.routeros_dns.dns-server (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.snmp.var.vrf (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.ip_address.routeros_ip_address.address (expand)" references: [module.ip_address.var.ip_address_config (expand) module.ip_address.var.ip_address_config (expand) module.ip_address.var.ip_address_config (expand) module.ip_address.var.ip_address_config (expand) module.ip_address.var.ip_address_config (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.snmp_community.routeros_snmp_community.community (expand)" references: [module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand) module.snmp_community.var.community_config (expand)]
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.dns_server.var.use_doh_server (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.bridges.var.bridge_config (expand)" references: []
2025-05-22T16:31:41.116+0300 [DEBUG] ReferenceTransformer: "module.ip_services (close)" references: []
2025-05-22T16:31:41.125+0300 [DEBUG] Starting graph walk: walkApply
2025-05-22T16:31:41.126+0300 [DEBUG] created provider logger: level=debug
2025-05-22T16:31:41.126+0300 [INFO]  provider: configuring client automatic mTLS
2025-05-22T16:31:41.131+0300 [DEBUG] provider: starting plugin: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 args=[".terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0"]
2025-05-22T16:31:41.131+0300 [DEBUG] provider: plugin started: path=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 pid=173182
2025-05-22T16:31:41.131+0300 [DEBUG] provider: waiting for RPC address: plugin=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0
2025-05-22T16:31:41.135+0300 [INFO]  provider.terraform-provider-routeros_v1.85.0: configuring server automatic mTLS: timestamp="2025-05-22T16:31:41.135+0300"
2025-05-22T16:31:41.148+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: plugin address: address=/tmp/plugin239995627 network=unix timestamp="2025-05-22T16:31:41.148+0300"
2025-05-22T16:31:41.148+0300 [DEBUG] provider: using plugin: version=5
2025-05-22T16:31:41.159+0300 [WARN]  ValidateProviderConfig from "provider[\"registry.terraform.io/terraform-routeros/routeros\"]" changed the config value, but that value is unused
2025-05-22T16:31:41.230+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: request body:  /system/resource/print: @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 timestamp="2025-05-22T16:31:41.230+0300"
2025-05-22T16:31:41.249+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: response body: !re @r1 [{`uptime` `11w3d3h32m12s`} {`version` `7.12.1 (stable)`} {`build-time` `Nov/17/2023 11:38:45`} {`factory-software` `6.46.4`} {`free-memory` `203706368`} {`total-memory` `268435456`} {`cpu` `MIPS 1004Kc V2.15`} {`cpu-count` `4`} {`cpu-frequency` `880`} {`cpu-load` `2`} {`free-hdd-space` `4292608`} {`total-hdd-space` `16777216`} {`write-sect-since-reboot` `23974`} {`write-sect-total` `37559`} {`architecture-name` `mmips`} {`board-name` `hEX S`} {`platform` `MikroTik`}]
!done @r1 []: tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros timestamp="2025-05-22T16:31:41.248+0300"
2025-05-22T16:31:41.249+0300 [INFO]  provider.terraform-provider-routeros_v1.85.0: RouterOS: 7.12.1: @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:40 timestamp="2025-05-22T16:31:41.249+0300"
routeros_ip_firewall_nat.test: Modifying... [id=*A]
2025-05-22T16:31:41.257+0300 [INFO]  Starting apply for routeros_ip_firewall_nat.test
2025-05-22T16:31:41.258+0300 [DEBUG] routeros_ip_firewall_nat.test: applying the planned Update change
2025-05-22T16:31:41.259+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: request body:  /ip/firewall/nat/print ?.id=*A: tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros timestamp="2025-05-22T16:31:41.259+0300"
2025-05-22T16:31:41.282+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: response body: !re @r2 [{`.id` `*A`} {`chain` `dstnat`} {`action` `dst-nat`} {`to-addresses` `1.2.3.4`} {`to-ports` `22`} {`protocol` `tcp`} {`dst-address` `1.2.3.4`} {`dst-port` `12345`} {`bytes` `0`} {`packets` `0`} {`invalid` `false`} {`dynamic` `false`} {`disabled` `true`} {`comment` `Test rule`}]
!done @r2 []: @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 tf_rpc=Configure timestamp="2025-05-22T16:31:41.282+0300"
2025-05-22T16:31:41.282+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: request body:  /ip/firewall/nat/set =comment=Test rule =in-interface=wan1.ether1 =dst-address= =to-addresses=1.2.3.4 =action=dst-nat =protocol=tcp =disabled=yes =to-ports=22 =chain=dstnat =dst-port=12345 =.id=*A: tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=af0d9c94-839c-9083-43c2-b357e470dab3 timestamp="2025-05-22T16:31:41.282+0300"
2025-05-22T16:31:41.300+0300 [DEBUG] provider.terraform-provider-routeros_v1.85.0: An error was encountered while sending a PATCH request to the API: from RouterOS device: value of range expects range of ip addresses: @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=fd299a63-2ea6-a52a-4006-e912e1dde104 tf_resource_type=routeros_ip_firewall_nat tf_rpc=ApplyResourceChange timestamp="2025-05-22T16:31:41.300+0300"
2025-05-22T16:31:41.301+0300 [ERROR] provider.terraform-provider-routeros_v1.85.0: Response contains error diagnostic: tf_rpc=ApplyResourceChange @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov5/internal/diag/diagnostics.go:58 diagnostic_summary="from RouterOS device: value of range expects range of ip addresses" tf_proto_version=5.8 tf_resource_type=routeros_ip_firewall_nat @module=sdk.proto diagnostic_detail="" diagnostic_severity=ERROR tf_provider_addr=terraform-routeros/routeros tf_req_id=fd299a63-2ea6-a52a-4006-e912e1dde104 timestamp="2025-05-22T16:31:41.301+0300"
2025-05-22T16:31:41.302+0300 [DEBUG] State storage *remote.State declined to persist a state snapshot
2025-05-22T16:31:41.302+0300 [ERROR] vertex "routeros_ip_firewall_nat.test" error: from RouterOS device: value of range expects range of ip addresses
2025-05-22T16:31:41.302+0300 [DEBUG] states/remote: state read serial is: 23; serial is: 23
2025-05-22T16:31:41.302+0300 [DEBUG] states/remote: state read lineage is: a8999863-b84f-4b34-6aa7-2f12c7026b30; lineage is: a8999863-b84f-4b34-6aa7-2f12c7026b30
2025-05-22T16:31:41.305+0300 [DEBUG] POST https://gitlab.ecom.tech/api/v4/projects/315/terraform/state/p02-oobgw-1?ID=0cf1a53e-f808-be6d-20b9-59b19490109d
╷
│ Error: from RouterOS device: value of range expects range of ip addresses
│ 
│   with routeros_ip_firewall_nat.test,
│   on locals-ip-firewall-nat.tf line 63, in resource "routeros_ip_firewall_nat" "test":
│   63: resource "routeros_ip_firewall_nat" "test" {
│ 
╵
2025-05-22T16:31:41.465+0300 [DEBUG] DELETE https://gitlab.ecom.tech/api/v4/projects/315/terraform/state/p02-oobgw-1/lock
2025-05-22T16:31:41.530+0300 [DEBUG] provider.stdio: received EOF, stopping recv loop: err="rpc error: code = Unavailable desc = error reading from server: EOF"
2025-05-22T16:31:41.531+0300 [INFO]  provider: plugin process exited: plugin=.terraform/providers/registry.terraform.io/terraform-routeros/routeros/1.85.0/linux_amd64/terraform-provider-routeros_v1.85.0 id=173182
2025-05-22T16:31:41.531+0300 [DEBUG] provider: plugin exited
```
**Stack Trace**
If applicable, add the stack trace the crash produced.

**Additional context**
Add any other context about the problem here.

--- a/routeros/resource_ip_firewall_filter.go
+++ b/routeros/resource_ip_firewall_filter.go
@@ -15,8 +15,9 @@ func ResourceIPFirewallFilter() *schema.Resource {
 		MetaResourcePath: PropResourcePath("/ip/firewall/filter"),
 		MetaId:           PropId(Id),
 		MetaSkipFields:   PropSkipFields("bytes", "packets"),
-		MetaSetUnsetFields: PropSetUnsetFields("dst_address_list", "src_address_list", "in_interface", "in_interface_list",
-			"out_interface", "out_interface_list", "in_bridge_port_list", "out_bridge_port_list", "protocol"),
+		MetaSetUnsetFields: PropSetUnsetFields("dst_address", "dst_address_list", "src_address", "src_address_list",
+			"in_interface", "in_interface_list", "out_interface", "out_interface_list", "in_bridge_port_list",
+			"out_bridge_port_list", "protocol"),
 
 		"action": {
 			Type:        schema.TypeString,

--- a/routeros/resource_ip_firewall_mangle.go
+++ b/routeros/resource_ip_firewall_mangle.go
@@ -30,8 +30,9 @@ func ResourceIPFirewallMangle() *schema.Resource {
 		MetaResourcePath: PropResourcePath("/ip/firewall/mangle"),
 		MetaId:           PropId(Id),
 		MetaSkipFields:   PropSkipFields("bytes", "packets"),
-		MetaSetUnsetFields: PropSetUnsetFields("dst_address_list", "src_address_list", "in_interface", "in_interface_list",
-			"out_interface", "out_interface_list", "in_bridge_port_list", "out_bridge_port_list"),
+		MetaSetUnsetFields: PropSetUnsetFields("dst_address", "dst_address_list", "src_address", "src_address_list",
+			"in_interface", "in_interface_list", "out_interface", "out_interface_list", "in_bridge_port_list",
+			"out_bridge_port_list"),
 
 		"action": {
 			Type:        schema.TypeString,

--- a/routeros/resource_ip_firewall_nat.go
+++ b/routeros/resource_ip_firewall_nat.go
@@ -32,8 +32,9 @@ func ResourceIPFirewallNat() *schema.Resource {
 		MetaResourcePath: PropResourcePath("/ip/firewall/nat"),
 		MetaId:           PropId(Id),
 		MetaSkipFields:   PropSkipFields("bytes", "packets"),
-		MetaSetUnsetFields: PropSetUnsetFields("dst_address_list", "src_address_list", "in_interface", "in_interface_list",
-			"out_interface", "out_interface_list", "in_bridge_port_list", "out_bridge_port_list"),
+		MetaSetUnsetFields: PropSetUnsetFields("dst_address", "dst_address_list", "src_address", "src_address_list",
+			"in_interface", "in_interface_list", "out_interface", "out_interface_list", "in_bridge_port_list",
+			"out_bridge_port_list"),
 
 		"action": {
 			Type:        schema.TypeString,

--- a/routeros/resource_ip_firewall_raw.go
+++ b/routeros/resource_ip_firewall_raw.go
@@ -31,8 +31,9 @@ func ResourceIPFirewallRaw() *schema.Resource {
 		MetaResourcePath: PropResourcePath("/ip/firewall/raw"),
 		MetaId:           PropId(Id),
 		MetaSkipFields:   PropSkipFields("bytes", "packets"),
-		MetaSetUnsetFields: PropSetUnsetFields("dst_address_list", "src_address_list", "in_interface", "in_interface_list",
-			"out_interface", "out_interface_list", "in_bridge_port_list", "out_bridge_port_list", "protocol"),
+		MetaSetUnsetFields: PropSetUnsetFields("dst_address", "dst_address_list", "src_address", "src_address_list",
+			"in_interface", "in_interface_list", "out_interface", "out_interface_list", "in_bridge_port_list",
+			"out_bridge_port_list", "protocol"),
 
 		"action": {
 			Type:        schema.TypeString,

